habitat-hitl/habitat_hitl/core/text_drawer.py

```diff
@@ -12,7 +12,8 @@
 import magnum as mn
 from magnum import shaders, text
 
-DEFAULT_FONT_PATH = "./fonts/ProggyClean.ttf"
+DEFAULT_FONT_PATH = "./fonts/IBMPlexSans-Regular.ttf"
```

Comment:
Since you've moved these defaults to the config, can we get rid of these here, for clarity/simplicity? I am totally fine with HITL apps absolutely requiring a proper hitl_defaults config (it's not optional).